README.md

```diff
@@ -487,6 +487,119 @@ $this->assertNotEquals(QueuedJob::STATUS_BROKEN, $descriptor->JobStatus);
 
 For example, this code snippet runs the job and checks if the job ended up in a non-broken state.
 
+## Special job variables
+
+It's good to be aware of special variables which should be used in your job implementation.
+
+* `totalSteps` (integer) - defaults to `0`, maps to `TotalSteps` DB column
+* `currentStep` (integer) - defaults to `0`, maps to `StepsProcessed` DB column
+* `isComplete` (boolean) - defaults to `false`, related to `JobStatus` DB column
+
+See `copyJobToDescriptor` for more details on the mapping between `Job` and `JobDescriptor`.
+
+### Total steps
+
```

Comment:
It would be good to have an anchor link to the other part of the readme that explains what `steps` are.

Reply:
Move this whole section closer to health checking as it's a related topic.